src/foam/u2/ActionView.js

```diff
@@ -346,7 +350,7 @@ foam.CLASS({
         this.attrs({ name: this.action.name });
 
         this.enableClass(this.myClass('unavailable'), this.action.createIsAvailable$(this.__context__, this.data), true);
-        this.attrs({ disabled: this.action.createIsEnabled$(this.__context__, this.data).map((e) => e ? false : 'disabled') });
+        this.attrs({ disabled: this.disabled ? 'disabled' : this.action.createIsEnabled$(this.__context__, this.data).map((e) => e ? false : 'disabled') });
```

Comment:
This means that we only hook up the slot if disabled was not set on first render.  Instead you should create a new slot which computes the disabled attribute based upon action.createIsEnable$ and this.disabled$